Source/WebCore/bindings/scripts/CodeGeneratorJS.pm

```diff
@@ -5376,17 +5376,11 @@ extern "C" { extern void (*const ${vtableRefWin}[])(); }
 #else
 extern "C" { extern void* ${vtableNameGnu}[]; }
 #endif
-#endif
-
-END
-
-        push(@implContent, "JSC::JSValue toJSNewlyCreated(JSC::JSGlobalObject*, JSDOMGlobalObject* globalObject, Ref<$implType>&& impl)\n");
-        push(@implContent, "{\n");
-        push(@implContent, <<END) if $vtableNameGnu;
-
-    if constexpr (std::is_polymorphic_v<${implType}>) {
-#if ENABLE(BINDING_INTEGRITY)
-        const void* actualVTablePointer = getVTablePointer(impl.ptr());
+    // Force the type being checked to be a dependent type so we can rely on
```

Comment:
I don't think it's necessary to add this comment to each generated cpp file.  It's reasonably clear what's going on, and it'll very soon be a compiler error if anyone does anything else.

Reply:
There are other generated comments so I really wasn't sure whether yet more comments was worth it or not. Will just make the comment in the generator.